### PR TITLE
Translating Community Page

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
An outcome of Google Summer of Code 2021
Author- Sanjay Singh Rajpoot @SanjaySinghRajpoot
Mentor - Aditya Rana @ranaaditya

### CODE REVIEW

File: src/data/hi.yml [hunk 1]
```diff
@@ -453,24 +453,38 @@ libraries:
...
+     पहचान और अभिव्यक्ति, यौन अभिविन्यास, जाति, जातीयता, भाषा,
+     न्यूरो-प्रकार, आकार, क्षमता, वर्ग, धर्म, संस्कृति, उपसंस्कृति, राजनीतिक
+     राय, आयु, कौशल स्तर, व्यवसाय और पृष्ठभूमि। हम मानते हैं कि
+     हर किसी के पास सक्रिय रूप से समय, वित्तीय साधन या क्षमता नहीं है
+     भाग लेते हैं, लेकिन हम सभी प्रकार की भागीदारी को पहचानते हैं और प्रोत्साहित करते हैं। हम
+     पहुंच और सशक्तिकरण को सुगम और बढ़ावा देना। हम सब शिक्षार्थी हैं।
+  community-statement3: >-
+     हमें ये हैशटैग पसंद हैं: #noCodeSnobs (क्योंकि हम समुदाय को महत्व देते हैं
+     दक्षता), #newKidLove (क्योंकि हम सभी ने कहीं न कहीं शुरुआत की है), #unassumeCore
+     (क्योंकि हम ज्ञान को ग्रहण नहीं करते हैं), और #BlackLivesMatter (क्योंकि .)
```
Comment (ranaaditya) [inline src/data/hi.yml]: For #blivesmatter tagg please check how to write it in a polite and gentle way

File: src/data/hi.yml [hunk 2]
```diff
@@ -479,32 +493,32 @@ community:
...
   develop-title: "विकास करना।"
   develop1: "GitHub"
-  develop2: " मुख्य स्थान है जहां कोड एकत्र किया जाता है, मुद्दों को प्रलेखित किया जाता है, और कोड के बारे में चर्चा होती है। आरंभ करने के लिए "
+  develop2: " मुख्य स्थान है जहां कोड एकत्र किया जाता है, मुद्दों का दस्तावेजीकरण किया जाता है, और कोड के बारे में चर्चा की जाती है। आरंभ करने के लिए "
   develop3: " विकास शिक्षण"
   develop4: " देखें, या "
   develop5: "अपना स्वयं का लाइब्रेरी बनाएं।"
   document-title: "दस्तावेज़।"
-  document1: " दस्तावेज़ीकरण का शौक सभी को होता है। "
+  document1: " सभी को दस्तावेज़ीकरण पसंद है। मदद की जरूरत है "
```
Comment (ranaaditya) [inline src/data/hi.yml]: If this is related to docs or documentation please find a better word, I think there should be something better than "daataavez"

File: src/data/hi.yml [hunk 3]
```diff
@@ -479,32 +493,32 @@ community:
...
   donate-title: "दान देना।"
   donate1: " p5.js स्वतंत्र और खुला स्रोत है और कलाकारों द्वारा बनाया गया है। "
   donate2: "प्रोसेसिंग फाउंडेशन"
-  donate3: " को दान के माध्यम से p5.js के विकास में सहायता करें।"
-  contributors-conference-title: "योगदानकर्ता सम्मेलन"
-  contributors-conference1: "जबकि अधिकांश कार्य ऑनलाइन होते है, हम IRL भी आयोजित करते हैं। हमने पिट्सबर्ग, पीए में  "
-  contributors-conference2: " कार्नेगी मेलन विश्वविद्यालय  में योगदानकर्ताओं के दो सम्मेलन आयोजित किए। कलाकार, डिजाइनर, डेवलपर्स, शिक्षक p5.js परियोजना को आगे बढ़ाने के लिए एक साथ शामिल हुए। "
+  donate3: " को दान के माध्यम से p5.js के विकास में सहायता करें। "
+  contributors-conference-title: "p5.js योगदानकर्ता सम्मेलन"
+  contributors-conference1: "जबकि अधिकांश कार्य ऑनलाइन होते है, हम IRL भी आयोजित करते हैं। हमने पिट्सबर्ग, पीए में "
```
Comment (ranaaditya) [inline src/data/hi.yml]: For Pittsburgh, PA you can write full form of PA.
Actually it's  less common practice to abbreviate places name in hindi.

**Review by ranaaditya — CHANGES_REQUESTED** [review]

File: src/data/hi.yml (hunk 2 above)
Comment (SanjaySinghRajpoot) [reply src/data/hi.yml]: प्रलेखन is  one alternate word for documentation other than "daataavez". So should I change it to प्रलेखन

File: src/data/hi.yml (hunk 3 above)
Comment (SanjaySinghRajpoot) [reply src/data/hi.yml]: ok I will make it to Pittsburgh, Pennsylvania

Comment (ranaaditya) [reply src/data/hi.yml]: Better

**Review by ranaaditya — APPROVED** [review]

**Pull request merged**